.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug] "
+labels: [':bug: Bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: pluginName
+    attributes:
+      label: Plugin Name and Version
```

Comment:
remove "and Version" from label please

Reply:
You missed this!

Reply:
Oops! My bad, missed pushing the commit and logged of 🙈